gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java

```diff
@@ -15,266 +15,112 @@
+        return configuration.getProperty("jetty.host", "0.0.0.0");
     }
 
-    public int getAcceptors() {
-        return acceptors;
+    public int getHttpPort() {
+        return configuration.getProperty("jetty.port", Integer.class, 8093);
     }
 
-    public void setAcceptors(int acceptors) {
-        this.acceptors = acceptors;
+    public int getAcceptors() {
+        return configuration.getProperty("jetty.acceptors", Integer.class, -1);
     }
 
     public int getSelectors() {
-        return selectors;
-    }
-
-    public void setSelectors(int selectors) {
-        this.selectors = selectors;
+        return configuration.getProperty("jetty.selectors", Integer.class, -1);
     }
 
     public int getPoolMinThreads() {
-        return poolMinThreads;
-    }
-
-    public void setPoolMinThreads(int poolMinThreads) {
-        this.poolMinThreads = poolMinThreads;
+        return configuration.getProperty("jetty.pool.minThreads", Integer.class, 10);
     }
 
     public int getPoolMaxThreads() {
-        return poolMaxThreads;
-    }
-
-    public void setPoolMaxThreads(int poolMaxThreads) {
-        this.poolMaxThreads = poolMaxThreads;
+        return configuration.getProperty("jetty.pool.maxThreads", Integer.class, 200);
     }
 
     public boolean isJmxEnabled() {
-        return jmxEnabled;
-    }
-
-    public void setJmxEnabled(boolean jmxEnabled) {
-        this.jmxEnabled = jmxEnabled;
+        return configuration.getProperty("jetty.jmx", Boolean.class, false);
     }
 
     public int getIdleTimeout() {
-        return idleTimeout;
-    }
-
-    public void setIdleTimeout(int idleTimeout) {
-        this.idleTimeout = idleTimeout;
+        return configuration.getProperty("jetty.idleTimeout", Integer.class, 30000);
     }
 
     public boolean isStatisticsEnabled() {
-        return statisticsEnabled;
-    }
-
-    public void setStatisticsEnabled(boolean statisticsEnabled) {
-        this.statisticsEnabled = statisticsEnabled;
+        return configuration.getProperty("jetty.statistics", Boolean.class, false);
     }
 
     public int getPoolIdleTimeout() {
-        return poolIdleTimeout;
-    }
-
-    public void setPoolIdleTimeout(int poolIdleTimeout) {
-        this.poolIdleTimeout = poolIdleTimeout;
+        return configuration.getProperty("jetty.pool.idleTimeout", Integer.class, 60000);
     }
 
     public int getPoolQueueSize() {
-        return poolQueueSize;
-    }
-
-    public void setPoolQueueSize(int poolQueueSize) {
-        this.poolQueueSize = poolQueueSize;
+        return configuration.getProperty("jetty.pool.queueSize", Integer.class, 6000);
     }
 
     public boolean isAccessLogEnabled() {
-        return accessLogEnabled;
-    }
-
-    public void setAccessLogEnabled(boolean accessLogEnabled) {
-        this.accessLogEnabled = accessLogEnabled;
+        return configuration.getProperty("jetty.accesslog.enabled", Boolean.class, true);
     }
 
     public String getAccessLogPath() {
```

Comment:
I suppose this configuration was done using the setter, right ?

Reply:
I just turned:

```java
    @Value("${jetty.accesslog.path:${gravitee.home}/logs/gravitee_accesslog_yyyy_mm_dd.log}")
    private String accessLogPath;
```

into the corresponding `configuration.getProperty(...) `